### PR TITLE
Info text

### PR DESCRIPTION
**Pull request by whataleen**
Hi I am happy I was able to get it on here, But I am having trouble making the button look pretty without changing the other buttons for some reason. I just need a little pointer so I can make it better for the group please and thank you!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -20,6 +20,8 @@
   <!-- Main styles sheet -->
   <link rel="stylesheet" href="assets/css/style.css" />
   <title>Quarantine Match</title>
+  <p id="foo">click on this element.</p>
```
Comment (elogonme) [inline index.html:23]: What is this p element for? If you do not use it, please remove it

File: index.html [hunk 2]
```diff
@@ -28,7 +30,14 @@
...
       <h1>Quarantine Match</h1>
-      <p>info-text</p>
+
+      <button onclick="myFunction()">Game Rules</button>
+
+      <script>
+        function myFunction() {
+          alert('Enter your name and D.O.B. in entry one and your crush name and D.O.B in entry two, you do not need crush D.O.B, once entered click FIND to see results ');
+        }
+      </script>
```
Comment (elogonme) [inline index.html:41]: We have kept all js script files separately. We need to stay consistent, so if you have any javascript it has to be in main.js or in separate js file.

File: index.html [hunk 3]
```diff
@@ -28,7 +30,14 @@
     <div class="center-align">
       <!-- Page Content goes here -->
       <h1>Quarantine Match</h1>
-      <p>info-text</p>
+
+      <button onclick="myFunction()">Game Rules</button>
```
Comment (elogonme) [inline index.html:34]: I think we do not need a Game Rules button. Just put the text itself on the page. in p tag or div tag. Also as per project requirements we cannot use alert to display text. We need to use html elements or Modal. 
But just text on the page in p, div or h2 tags will be fine.

Comment (elogonme) [reply index.html:34]: And for making buttons same we use materialize framework (like bootstrap) on this project to have all elements in same theme. It was a part of project requirement too to learn new framework and we have chosen materialize.

File: index.html (hunk 2 above)
Comment (elogonme) [reply index.html:41]: So let me know if you can do changes or I can do it too.

File: index.html (hunk 1 above)
Comment (whataleen) [reply index.html:23]: Ok thank you

Comment (whataleen) [reply index.html:23]: I had a question I was changing the button style, It was changing the other buttons, what is the best way to make it right please and thank you

Comment (whataleen) [reply index.html:23]: Hi is this for me? , I am looking at my code and I have no p element not sure why you are seeing this?

File: index.html (hunk 2 above)
Comment (whataleen) [reply index.html:41]: so you would like me to move to the js I see, OK i can do this after class 👍🏽

**Pull request closed without merging**